service/src/main/java/uk/gov/hmcts/reform/fpl/model/CaseData.java

```diff
@@ -621,4 +620,13 @@ public boolean hasFutureHearing(List<Element<HearingBooking>> hearingBookings) {
             .filter(hearingBooking -> !hearingBooking.getValue().startsAfterToday())
             .collect(toList());
     }
+
+    private final Object cmoToReviewList;
+    private final ReviewDecision reviewCMODecision;
+    private final String numDraftCMOs;
+    private final List<Element<uk.gov.hmcts.reform.fpl.model.order.CaseManagementOrder>> sealedCMOs;
+
+    public List<Element<uk.gov.hmcts.reform.fpl.model.order.CaseManagementOrder>> getSealedCMOs() {
```

Comment:
i think it would be easier to import this package and have fully qualified name for old, it would cleanup story easier, but it is really minor

Reply:
will be fixed during cleanup story https://tools.hmcts.net/jira/browse/FPLA-2060